build/pxc-entrypoint.sh

```diff
@@ -302,8 +302,11 @@ if [ -z "$CLUSTER_JOIN" ] && [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 		fi
```

Comment:
it is also important to add skip-replica-start into this file
```
--- build/pxc-entrypoint.sh
+++ build/pxc-entrypoint.sh
@@ -148,6 +148,7 @@ function join {
 }

 MYSQL_VERSION=$(mysqld -V | awk '{print $3}' | awk -F'.' '{print $1"."$2}')
+MYSQL_PATCH_VERSION=$(mysqld -V | awk '{print $3}' | awk -F'.' '{print $3}' | awk -F'-' '{print $1}')

 # if vault secret file exists we assume we need to turn on encryption
 vault_secret="/etc/mysql/vault-keyring-secret/keyring_vault.conf"
@@ -179,6 +180,13 @@ fi
 grep -q "^progress=" $CFG && sed -i "s|^progress=.*|progress=1|" $CFG
 grep -q "^\[sst\]" "$CFG" || printf '[sst]\n' >>"$CFG"
 grep -q "^cpat=" "$CFG" || sed '/^\[sst\]/a cpat=.*\\.pem$\\|.*init\\.ok$\\|.*galera\\.cache$\\|.*wsrep_recovery_verbose\\.log$\\|.*readiness-check\\.sh$\\|.*liveness-check\\.s$
+if [[ "$MYSQL_VERSION" == '8.0' ]]; then
+       if [[ $MYSQL_PATCH_VERSION -ge 26 ]]; then
+               grep -q "^skip_replica_start=ON" "$CFG" || sed -i "/\[mysqld\]/a skip_replica_start=ON" $CFG
+       else
+               grep -q "^skip_slave_start=ON" "$CFG" || sed -i "/\[mysqld\]/a skip_slave_start=ON" $CFG
+       fi
+fi

 file_env 'XTRABACKUP_PASSWORD' 'xtrabackup' 'xtrabackup'
 file_env 'CLUSTERCHECK_PASSWORD' '' 'clustercheck'
 ```

Reply:
acabc84